code/modules/antagonists/pirate/pirate_roles.dm

```diff
@@ -94,3 +94,37 @@
 
 /obj/effect/mob_spawn/ghost_role/human/pirate/silverscale/gunner
 	rank = "Top-drawer"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/interdyne
+	name = "\improper Interdyne sleeper"
+	desc = "A surprisingly clean cryogenic sleeper. You can see your reflection on the sides!"
+	density = FALSE
+	you_are_text = "You are an Ex-Interdyne pharmacyst now turned space pirate."
+	flavour_text = "The station has refused to fund your research, so you will 'convince' them to donate to your charitable cause."
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	prompt_name = "An Ex-Interdyne employee"
+	outfit = /datum/outfit/pirate/interdyne
+	rank = "Pharmacist"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/interdyne/senior
+	rank = "Pharmacist Director"
+	outfit = /datum/outfit/pirate/interdyne/captain
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/interdyne/junior
+	rank = "Pharmacist"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/grey
+	name = "\improper Assistant sleeper"
+	desc = "A very dirty cryogenic sleeper. You're not sure if it even works."
+	density = FALSE
+	you_are_text = "You used to be a Nanotrasen assistant, until a riot gone awry. Now you wander space, ransacking any ships you come across!"
+	flavour_text = "There's nothing a toolbox can't whack in the head enough times to spill blood, or in this case money. Loot everything!"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	prompt_name = "An assistant gone loose"
+	outfit = /datum/outfit/pirate/grey
+	rank = "Tider"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/grey/shitter
```

Comment:
I also don't like this name, would prefer one of the suggestions the other user offered like Tidemaster

Reply:
the idea is that all 6 of them have the same amount of control as tiders unity under the same rank tidemaster would say there is someone above tiders which in tider spirit there is only tiders that tide under no rule or rank but sure I can change it to tidemaster